feature-libs/quote/components/details/overview/quote-details-overview.component.html

```diff
@@ -17,6 +17,8 @@ <h3 class="cx-status">
             getQuoteInformation(quoteDetails.name, quoteDetails.description)
               | async
           "
+          [truncateText]="true"
+          [charactersLimit]="30"
```

Comment:
or even better would be to remove the fixed character limit and truncate after one or two lines, so the styling adapts to the device. see also https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

Reply:
@Uli-Tiger, your suggestion make sense on the mobile widgets but not on the desktop. Let me share the screenshot:
![image](https://github.com/SAP/spartacus/assets/61147963/75f5c5cd-afd8-4133-8681-7d5f56d1b38d)
As you could see in the example with 60 characters there is too much empty space in the last 2 card tiles.
The UI designers said us to avoid this.

Reply:
It would be possible to configure the number of characters for the different devices, but I don't see any benefit in doing so.
Then the question remains why truncation is used when all the text could be displayed, but all the text explodes the size of the card tile.